trinity/protocol/eth/proto.py

```diff
@@ -115,7 +122,12 @@ def send_block_bodies(self, blocks: List[BlockBody]) -> None:
         header, body = cmd.encode(blocks)
         self.send(header, body)
 
-    def send_get_receipts(self, block_hashes: List[Hash32]) -> None:
+    def send_get_receipts(self,
+                          request: Union[ReceiptsRequest, Tuple[Hash32, ...]]) -> None:
```

Comment:
Is this type Union (and the `send_get_node_data` one) just to make testing easier? If so, it might be nice to simplify this to only `ReceiptsRequest` and do a bit more setup work in the tests. (also get to drop the `isinstance` which is occasionally slow, in tight loops)